internal/release/scripts/prep-release.sh

```diff
@@ -2,12 +2,27 @@
 
 # Helper script to prepare a release for the Go SDK.
 
-# Read the current build number from version-build to make sure the build number has been updated
-current_build_number=$(< internal/release/version-build)
+# Read current build and version for backup as well as comparison to new build
+current_build=$(< internal/release/version-build)
+current_version=$(< internal/release/version)
```

Comment:
Could we also do a check on the version string, just to check that it's not the same as the previous version string, just like we do for build number?

Reply:
Done